Example/Tests/Doubles/BitmovinPlayerTestDouble.swift

```diff
@@ -243,12 +243,15 @@ class BitmovinPlayerTestDouble: BitmovinPlayerStub, TestDoubleDataSource {
 }
 
 class BitmovinPlayerStub: NSObject, Player {
+    var latency: BitmovinPlayerCore.LatencyApi
```

Comment:
We should follow the pattern used in this class.

```suggestion
    var latency: BitmovinPlayerCore.LatencyApi {
        player.latency
    }
```

Then the initialization can also be removed in line 254.

Reply:
Good suggestion, I've applied the change